Board.py

```diff
@@ -1,11 +1,163 @@
-from checker import Checker
+from enum import Enum
+
+ROWS = 8
+COLS = 8
+
+
+class Player(Enum):
+    player1 = 0
+    player2 = 1
+
+
+class Piece(Enum):
+    basic = 0
+    king = 0
+
+
+def in_bounds(x, y):
+    return 0 <= x < COLS and 0 <= y < ROWS
+
+
+def is_jump(start_x, end_x):
+    return abs(end_x - start_x) > 1
 
 
 class Board:
+
     def __init__(self):
-        rows, cols = 8, 8
-        self.board = [['0' for _ in range(cols)] for _ in range(rows)]
+        self.player_one_pieces = []
+        self.player_two_pieces = []
+        for i in range(3):
+            for z in range(4):
+                checker_x = i
+                checker_y = z * 2 + (i + 1) % 2
+                temp_piece = self.generate_checker(checker_x, checker_y, Player.player1)
+                self.player_one_pieces.append(temp_piece)
+
+                checker_x = i + 5
+                checker_y = z * 2 + i % 2
+                temp_piece = self.generate_checker(checker_x, checker_y, Player.player2)
+                self.player_two_pieces.append(temp_piece)
+
+    def move(self, move):  # move tuple ==> (start_x, start_y, end_x, end_y, Player)
+        start_x = move[0]
+        start_y = move[1]
+        end_x = move[2]
+        end_y = move[3]
+        player = move[4]
+
+        if is_jump(start_x, end_x):
+            self.remove_checker(int((start_x + end_x) / 2), int((start_y + end_y) / 2))  # remove piece
```

Comment:
`int((start_x + end_x) / 2), int((start_y + end_y) / 2)` should be its own one line independent helper function. so the code should look like `self.remove_checker(get_checker_to_remove(start_x, end_x))` or something like that